TextureSwiftSupport/SpecBuilder.swift

```diff
@@ -151,6 +151,19 @@ public struct MultiLayout : _ASLayoutElementType {
   }
 }
 
+public struct AnyLayout : _ASLayoutElementType {
+  
+  public let content: _ASLayoutElementType
+  
+  public init(_ content: () -> _ASLayoutElementType) {
```

Comment:
Is it okay to make this closure `@ASLayoutSpecBuilder` ?

Reply:
wow, I forgot it.
It would be better that uses Builder.
I'll update it.
Thank you!